grakn/service/Session/Concept/ConceptFactory.py

```diff
@@ -17,34 +17,52 @@
 # under the License.
 #
 
-from grakn.service.Session.Concept import BaseTypeMapping
-from grakn.service.Session.Concept.Concept import EntityType, RelationType, AttributeType, Role, Rule, Entity, Relation, Attribute, Type
-
+#from grakn.service.Session.Concept import BaseTypeMapping
+import BaseTypeMapping
```

Comment:
@adammitchelldev this import doesn't quite look fine to me :thinking: 
have you tested it?

Reply:
Yes, it does work fine but it can be reverted now I think.